flo-freezer/src/main/java/com/spotify/flo/freezer/PersistingContext.java

```diff
@@ -95,41 +97,50 @@ public PersistingContext(Path basePath, EvalContext delegate) {
     return promise.value();
   }
 
-  public static void serialize(Object object, Path file) throws Exception{
-    final Kryo kryo = new Kryo();
-    PackageRegistrar.all().apply(kryo);
-    kryo.register(java.lang.invoke.SerializedLambda.class);
-    kryo.register(ClosureSerializer.Closure.class, new ClosureSerializer());
-    kryo.addDefaultSerializer(java.lang.Throwable.class, new JavaSerializer());
-    kryo.getFieldSerializerConfig().setIgnoreSyntheticFields(false);
-
-    if (Files.exists(file)) {
-      throw new RuntimeException("File " + file + " already exists");
-    }
-
+  public static void serialize(Object object, Path file) throws Exception {
     try {
-      try (Output output = new Output(newOutputStream(file, WRITE, CREATE_NEW))) {
-        kryo.writeClassAndObject(output, object);
-      }
+      serialize(object, Files.newOutputStream(file, WRITE, CREATE_NEW));
```

Comment:
`newOutputStream` has been statically imported, the same for `newInputStream`.

Reply:
@labianchin fix this? i will give +1 then. :)

Reply:
I removed the static imports. Or should I use them?

Reply:
Ah, OK, either way. I didn't see that because it didn't close this comment. Sorry.